renovate.json

```
@@ -26,6 +26,11 @@
       "groupName": "all BBC non-major dependencies",
       "updateTypes": ["minor", "patch"],
       "groupSlug": "psammead-minor-patch"
+    },
+    {
+      "packageNames": ["@bbc/psammead"],
```

This config didn't seem to work when I tried it on my fork: https://github.com/andrewscfc/simorgh/commit/a084bbdad62d27e594e7f758a51db1aefd964f26  with dependabot enabled.

It may be there is a slash in regext that needs escaping, this _may_ work?
```suggestion
      "packageNames": ["psammead"],
```